container/features/src/main/resources/features.xml

```diff
@@ -1121,9 +1121,17 @@
       <bundle>mvn:org.opennms.features.telemetry.adapters/org.opennms.features.telemetry.adapters.collection/${project.version}</bundle>
       <bundle>mvn:org.opennms.features.telemetry.adapters/org.opennms.features.telemetry.adapters.jti/${project.version}</bundle>
     </feature>
+    
+    <feature name="opennms-telemetry-nxos" description="OpenNMS :: Telemetry :: NXOS" version="${project.version}">
+      <feature>opennms-collection-api</feature>
+      <feature version="${guavaVersion}">guava</feature>
+      <bundle>mvn:com.google.protobuf/protobuf-java/3.4.0</bundle>
+      <bundle>mvn:org.opennms.features.telemetry.adapters/org.opennms.features.telemetry.adapters.collection/${project.version}</bundle>
```

Comment:
Can we break out the `collection` adapter and it's dependencies into a `opennms-telemetry-collection` feature and depend on this for both JTI and NX-OS?